packages/admin-next/dashboard/src/routes/api-key-management/api-key-management-detail/components/api-key-sales-channel-section/api-key-sales-channel-section.tsx

```diff
@@ -0,0 +1,370 @@
+                              header.getContext()
+                            )}
+                          </Table.HeaderCell>
+                        )
+                      })}
+                    </Table.Row>
+                  )
+                })}
+              </Table.Header>
+              <Table.Body className="border-b-0">
+                {table.getRowModel().rows.map((row) => (
+                  <Table.Row
+                    key={row.id}
+                    className={clx(
+                      "transition-fg cursor-pointer [&_td:last-of-type]:w-[1%] [&_td:last-of-type]:whitespace-nowrap",
+                      {
+                        "bg-ui-bg-highlight hover:bg-ui-bg-highlight-hover":
+                          row.getIsSelected(),
+                      }
+                    )}
+                    onClick={() =>
+                      navigate(`/settings/sales-channels/${row.original.id}`)
+                    }
+                  >
+                    {row.getVisibleCells().map((cell) => (
+                      <Table.Cell key={cell.id}>
+                        {flexRender(
+                          cell.column.columnDef.cell,
+                          cell.getContext()
+                        )}
+                      </Table.Cell>
+                    ))}
+                  </Table.Row>
+                ))}
+              </Table.Body>
+            </Table>
+          ) : (
+            <NoResults />
+          )}
+          <LocalizedTablePagination
+            canNextPage={table.getCanNextPage()}
+            canPreviousPage={table.getCanPreviousPage()}
+            nextPage={table.nextPage}
+            previousPage={table.previousPage}
+            count={count ?? 0}
+            pageIndex={pageIndex}
+            pageCount={table.getPageCount()}
+            pageSize={PAGE_SIZE}
+          />
+          <CommandBar open={!!Object.keys(rowSelection).length}>
+            <CommandBar.Bar>
+              <CommandBar.Value>
+                {t("general.countSelected", {
+                  count: Object.keys(rowSelection).length,
+                })}
+              </CommandBar.Value>
+              <CommandBar.Seperator />
+              <CommandBar.Command
+                action={handleRemove}
+                shortcut="r"
+                label={t("general.remove")}
+              />
+            </CommandBar.Bar>
+          </CommandBar>
+        </div>
+      )}
+    </Container>
+  )
+}
+
+const SalesChannelActions = ({
+  salesChannel,
+  apiKey,
+}: {
+  salesChannel: SalesChannel
+  apiKey: string
+}) => {
+  const { t } = useTranslation()
+  const prompt = usePrompt()
+
+  const { mutateAsync } = useAdminRemovePublishableKeySalesChannelsBatch(apiKey)
+
+  const handleDelete = async () => {
+    const res = await prompt({
+      title: t("general.areYouSure"),
+      description: t("locations.removeSalesChannelsWarning", { count: 1 }),
```

Comment:
todo: Message should be specific to API keys. Showing "locations" right now:
![CleanShot 2024-01-22 at 20 04 39](https://github.com/medusajs/medusa/assets/59018053/a914ec14-44c3-4388-aab5-86c5f3d6cd95)